$OpenBSD: patch-Makefile,v 1.9 2020/12/07 05:19:19 bket Exp $

Fix empty version in .pc file. Taken from
https://github.com/Cyan4973/xxHash/pull/442.

Don't create symbolic links pointing to the shared lib.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -437,7 +437,7 @@ libxxhash.pc: libxxhash.pc.in
           -e 's|@EXECPREFIX@|$(PCEXECDIR)|' \
           -e 's|@LIBDIR@|$(PCLIBDIR)|' \
           -e 's|@INCLUDEDIR@|$(PCINCDIR)|' \
-          -e 's|@VERSION@|$(VERSION)|' \
+          -e 's|@VERSION@|$(LIBVER)|' \
           $< > $@
 
 
@@ -447,8 +447,6 @@ install: lib libxxhash.pc xxhsum  ## install libraries
 	$(Q)$(INSTALL) -d -m 755 $(DESTDIR)$(LIBDIR)
 	$(Q)$(INSTALL_DATA) libxxhash.a $(DESTDIR)$(LIBDIR)
 	$(Q)$(INSTALL_PROGRAM) $(LIBXXH) $(DESTDIR)$(LIBDIR)
-	$(Q)ln -sf $(LIBXXH) $(DESTDIR)$(LIBDIR)/libxxhash.$(SHARED_EXT_MAJOR)
-	$(Q)ln -sf $(LIBXXH) $(DESTDIR)$(LIBDIR)/libxxhash.$(SHARED_EXT)
 	$(Q)$(INSTALL) -d -m 755 $(DESTDIR)$(INCLUDEDIR)   # includes
 	$(Q)$(INSTALL_DATA) xxhash.h $(DESTDIR)$(INCLUDEDIR)
 	$(Q)$(INSTALL_DATA) xxh3.h $(DESTDIR)$(INCLUDEDIR) # for compatibility, will be removed in v0.9.0
